src/quo/components/settings/settings_item/style.cljs

```diff
@@ -28,9 +28,10 @@
    :justify-content   :flex-start})
 
 (defn image-container
-  [image tag description]
+  [description tag image]
   {:height     (if (= image :icon-avatar) 32 20)
-   :margin-top (if (or tag description) 1 0)})
+   :margin-top (if (or tag description) 1 0)
+   :margin-left (when (nil? image) 4)})
```

Comment:
This fixes a small bug where the call-site was passing these arguments in the wrong order.